src/components/PizzaForm.js

```diff
@@ -0,0 +1,201 @@
+import React, { useState } from "react";
+import axios from "axios";
+import * as yup from "yup";
+
+// A name text input field
+//  Validation for name - name must be at least 2 characters
+//  A dropdown for pizza size
+//  A checklist for toppings - at least 4 (hint: name each separately!)
+//  Text input for special instructions
+//  An Add to Order button that submits form and returns a database record of name, size, toppings and special instructions
+
+const PizzaForm = () => {
+  const [formStart, setFormStart] = useState({
+    name: "",
+    sizes: {
+      small: false,
+      medium: false,
+      large: false,
+    },
+    toppings: {
+      pepperoni: false,
+      mushroom: false,
+      anchovie: false,
+      bacon: false,
+    },
+    instructions: "",
+  });
+
+  const handleChanges = (e) => {
+    if (e.target.type === "checkbox") {
```

Comment:
From a code style perspective this isn't ideal because if you added more checkboxes, your code would still be assuming that you're modifying the toppings. I would recommend basing your if-else on something such as "is the target name one of the toppings?" instead. It's more code, but easier to understand and easier to modify in the future.